repytah/assemble.py

```diff
@@ -83,14 +83,11 @@ def breakup_overlaps_by_intersect(input_pattern_obj, bw_vec, thresh_bw):
             One less than the smallest allowable repeat length.
 
     Returns:
-        pattern_no_overlaps (np.ndarray):
-            Binary matrix with 1's where repeats of essential structure
-            components begin.
-
-        pattern_no_overlaps_key (np.ndarray):
-            Vector containing the lengths of the repeats of essential structure
-            components in pattern_no_overlaps.
-            
+        A tuple (pattern_no_overlaps, pattern_no_overlaps_key) where
+        pattern_no_overlaps is a np.ndarray binary matrix with 1's where repeats
```

Comment:
I am wondering if the format could look like the format in transform.py where we say those two variables are np.ndarray binary matrix and start a new line to introduce the variables. Just for the ease of reading.